sources/lib/Twig/Extension/ProfilerExtension.php

```diff
@@ -9,8 +9,6 @@
  */
 namespace PommProject\PommBundle\Twig\Extension;
 
-use Symfony\Bundle\TwigBundle\Loader\FilesystemLoader;
```

Comment:
These changes is a part of PR #63.

The PR is merged in master, can you rebase this PR?